client/ayon_applications/manager.py

```diff
@@ -47,6 +89,160 @@ def __init__(self, studio_settings=None):
 
         self.refresh()
 
+    @staticmethod
+    def get_process_info_storage_location() -> Path:
+        """Get the path to process info storage.
+
+        Returns:
+            Path: Path to the process handlers storage.
+
+        """
+        storage_root = os.getenv("AYON_LAUNCHER_LOCAL_DIR")
+        if not storage_root:
+            msg = (
+                "Cannot determine process handlers storage location. "
+                "Environment variable 'AYON_LAUNCHER_LOCAL_DIR' is not set. "
+            )
+            raise RuntimeError(msg)
+        return Path(storage_root) / "process_handlers.db"
+
+    def _get_process_storage_connection(self) -> sqlite3.Connection:
+        """Store process handlers in the addon.
+
+        Returns:
+            sqlite3.Connection: Connection to the process handlers storage.
+
+        """
+        cnx = sqlite3.connect(self.get_process_info_storage_location())
+        cursor = cnx.cursor()
+        cursor.execute(
+            "CREATE TABLE IF NOT EXISTS process_info ("
+            "hash TEXT PRIMARY KEY, "
+            "name TEXT, "
+            "args TEXT DEFAULT NULL, "
+            "env TEXT DEFAULT NULL, "
+            "cwd TEXT DEFAULT NULL, "
+            "pid INTEGER DEFAULT NULL, "
+            "output_file TEXT DEFAULT NULL, "
+            "created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP, "
+            "site_id TEXT DEFAULT NULL"
```

Comment:
Why is site id used? The database is located at the same place where site id is defined...